documentation/modules/operators/con-topic-replication.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='ref-topic-replication-{context}']
+= Configuring topics for replication and scaling
```

Comment:
I think scaling is not the right term here since the partition counts might have many different roles.

```suggestion
= Configuring topics for replication and number of partitions
```